lib/scholar/metrics/distance.ex

```diff
@@ -194,16 +194,16 @@ defmodule Scholar.Metrics.Distance do
   """
   @spec manhattan(Nx.t(), Nx.t(), keyword()) :: Nx.t()
   deftransform manhattan(x, y, opts \\ []) do
-    nmanhattan(x, y, NimbleOptions.validate!(opts, @general_schema))
+    manhattan_n(x, y, NimbleOptions.validate!(opts, @general_schema))
   end
 
-  defnp nmanhattan(x, y, opts \\ []) do
+  defnp manhattan_n(x, y, opts \\ []) do
```

Comment:
For regular elixir function it's common to use the naming `def manhattan` and `defp do_manhattan` when "wrapping" a function into a more friendly API.

Note sure if we want to push a different convention for `defn`, but I think `_n` can be more surprising for an external reader.

If anyone has a strong opinion, please speak :)

Reply:
I started the `_n` exactly to signal that we are calling the actual numeric implementation. So I am in favor :D